etl/steps/data/garden/itopf/2023-05-18/oil_spills.meta.yml

```diff
@@ -0,0 +1,110 @@
+dataset:
+  title: Oil Spills
+  description: |-
+    ITOPF maintains a database of oil spills from tank vessels, including combined carriers, FPSOs and barges. This contains information on
```

Comment:
Nit - line breaks are weird, the output then looks slightly off
<img width="570" alt="image" src="https://github.com/owid/etl/assets/1550888/4b0b2542-e30d-4dce-ad74-a878c02aa48a">

Reply:
Line breaks in YAML are common headache unfortunately. Try to learn what different symbols for folded and multiline strings do (`>` and `|`), e.g. [here](https://quickref.me/yaml.html), then try it out and pick what you prefer.